### ISSUE TITLE
UR bugs: bugs in urscript are unreproducable

### ISSUE THREAD

**Issue by gavanderhoorn** [robust-rosin/robust#53]
The following bugs are issues in urscript programs:

 - https://github.com/robust-rosin/robust/blob/master/universal_robot/1e0904c.bug
 - https://github.com/robust-rosin/robust/blob/master/universal_robot/359a2e9.bug
 - https://github.com/robust-rosin/robust/blob/master/universal_robot/cda133d.bug

As we don't have any interpreter for URScript, we cannot really 'reproduce' them.

**Comment by gavanderhoorn** [comment]
Ping?

this is not dependend on the time machine, but in general this is unreproducable, at least L3.

**Comment by gavanderhoorn** [comment]
related: #45.

**Comment by wasowski** [comment]
Yeah, I tend to agree.  At L3 it is impossible, so we should probably somehow list them as "done" (blacklisted or whatever). If we fancy this, we may do L2 for them, or just not do them.
